src/cascadia/TerminalApp/App.cpp

```diff
@@ -916,7 +907,11 @@ namespace winrt::TerminalApp::implementation
         // Initialize the new tab
 
         // Create a Conhost connection based on the values in our settings object.
-        TerminalConnection::ITerminalConnection connection = TerminalConnection::ConhostConnection(settings.Commandline(), settings.StartingDirectory(), 30, 80, winrt::guid());
+        TerminalConnection::ITerminalConnection connection = TerminalConnection::ConhostConnection(settings.Commandline(),

```

Comment:
Can you just double check the spacing here? This looks horrifically badly spaced in GitHub's diff view: 
![image](https://user-images.githubusercontent.com/18221333/61245640-d1500900-a701-11e9-8f00-41d8b3a24c60.png)

Reply:
(and below)